G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/ExcelSwitchCommand.md

```diff
@@ -1,60 +1,36 @@
+|`id`| [integer](G1ANT.Language/G1ANT.Language/Structures/IntegerStructure.md) | yes |  | ID number of an Excel instance that will be activated |
+| `if`           | [bool](G1ANT.Language/G1ANT.Language/Structures/BooleanStructure.md) | no       | true                                                        | Executes the command only if a specified condition is true   |
+| `timeout`      | [timespan](G1ANT.Language/G1ANT.Language/Structures/TimeSpanStructure.md) | no       | [♥timeoutcommand](G1ANT.Language/G1ANT.Addon.Core/Variables/TimeoutCommandVariable.md) | Specifies time in milliseconds for G1ANT.Robot to wait for the command to be executed |
+| `errorcall`    | [procedure](G1ANT.Language/G1ANT.Language/Structures/ProcedureStructure.md) | no       |                                                             | Name of a procedure to call when the command throws an exception or when a given `timeout` expires |
+| `errorjump`    | [label](G1ANT.Language/G1ANT.Language/Structures/LabelStructure.md) | no       |                                                             | Name of the label to jump to when the command throws an exception or when a given `timeout` expires |
+| `errormessage` | [text](G1ANT.Language/G1ANT.Language/Structures/TextStructure.md) | no       |                                                             | A message that will be shown in case the command throws an exception or when a given `timeout` expires, and no `errorjump` argument is specified |
+| `errorresult`  | [variable](G1ANT.Language/G1ANT.Language/Structures/VariableStructure.md) | no       |                                                             | Name of a variable that will store the returned exception. The variable will be of [error](G1ANT.Language/G1ANT.Language/Structures/ErrorStructure.md) structure  |
 
-For more information about `if`, `timeout`, `errorjump` and `errormessage` arguments, please visit [Common Arguments](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Common-Arguments.md)  manual page.
+For more information about `if`, `timeout`, `errorcall`, `errorjump`, `errormessage` and `errorresult` arguments, see [Common Arguments](G1ANT.Manual/appendices/common-arguments.md) page.
 
-This command is contained in **G1ANT.Addon.MSOffice.dll**.
-See: [https://github.com/G1ANT-Robot/G1ANT.Addon.MSOffice](https://github.com/G1ANT-Robot/G1ANT.Addon.MSOffice)
+## Example
 
-**Example 1:**
+In order to use the `id` argument for the `excel.switch` command, you need to set an ID while using the `excel.open` command. The following example opens an empty Excel sheet, then an Excel file located on your Desktop. Both instances have their IDs stored in their respective `♥excel1` and `♥excel2` variables. Finally, the first ID is used to switch to the first Excel instance:
```

Comment:
You can't set an ID for opened by Robot excel instances. However, you can store the automatically set IDs in some variable using the result argument
so change this sentence:
```In order to use the `id` argument for the `excel.switch` command, you need to set an ID while using the `excel.open` command.```

Reply:
```suggestion
In order to use the `id` argument for the `excel.switch` command, you need to store an Excel instance ID in a variable while using the `excel.open` command. The following example opens an empty Excel sheet, then an Excel file located on your Desktop. Both instances have their IDs stored in their respective `♥excel1` and `♥excel2` variables. Finally, the first ID is used to switch to the first Excel instance:
```